_posts/2024-11-07-next-lts-3-15.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: 'Our next LTS will be Quarkus 3.15'
+date: 2024-07-11
+tags: release
+synopsis: 'Our next LTS (Long Term Support) Quarkus release will be Quarkus 3.15 released at the end of September.'
+author: gsmet
+---
+
+We started releasing LTS releases with Quarkus 3.2.
+LTS releases are supported significantly longer than our regular Quarkus releases.
+
+If you don't want to follow closely the development of Quarkus,
+and if you don't want to update regularly your applications to the latest Quarkus,
+LTS releases are the best choice for you.
+
+Our second LTS was Quarkus 3.8, released in February.
+
+It is our pleasure to announce that the next LTS will be Quarkus 3.15,
+that we plan to release at the end of September.
+
+For 3.15, we will follow the same strategy we inaugurated for 3.8,
+strategy that makes sure the LTS release is rock solid from the start.
+
+[IMPORTANT]
+====
+**tl;dr**
+
+Quarkus 3.15 will be our next LTS release.
+It will be the direct continuation of the `3.14` branch.
+
+If you contribute to Quarkus or the Quarkus Platform and need a feature in the next Quarkus LTS,
+make sure it has been merged in the https://github.com/quarkusio/quarkus[Quarkus repository] before August 13th included
+(the day before the `3.14.0.CR1` release).
+====
+
+== Quarkus 3.13
+
+Quarkus 3.13 will be a regular minor version of Quarkus.
+
+It will be released on July 31st.
+
+See our https://github.com/quarkusio/quarkus/wiki/Release-Planning[release schedule] for all the details.
+
+== Quarkus 3.14
+
+Quarkus 3.14 will be released on August 28th.
+
+We will branch Quarkus 3.14 from `main` when we release 3.14.0.CR1 on August 14th.
+
+After branching, `main` will host the development for Quarkus **3.16**.
+
+== Quarkus 3.15
+
+Quarkus 3.15 will be our next LTS version.
+It will be released on September 25th.
+
+This release will be the direct continuation of the 3.14 cycle and will actually get branched from the `3.14` branch.
+The focus for the 3.15 cycle will be on hardening 3.14 and fixing issues.
+It won't contain any new features.
+It might contain some additional component upgrades to fix CVEs or important bugs.
+
+**Consequently, and this is important:
+if you contribute to Quarkus or the Quarkus Platform and need a feature in the next Quarkus LTS,
+make sure it has been merged in the https://github.com/quarkusio/quarkus[Quarkus repository] before August 13th included
+(the day before the `3.14.0.CR1` release).**
```

Comment:
we should have something for Quarkiverse/non-quarkus participators too, in the original LTS (https://quarkus.io/blog/lts-releases/) we had something like:

```
Quarkus Extension Maintainers and Contributors

All of the above applies to Quarkus extension maintainers and contributors as well. In addition, we will be recommending that extension maintainers and contributors consider bug fixes and enhancements for LTS releases. This will ensure that LTS releases are as stable and robust as possible.

This means that extension maintainers and contributors will need to consider having branches and versioning in place for LTS releases.
```

Reply:
I added something along these lines in the 3.15 paragraph.